### PR TITLE
Pricing table

### PR DESCRIPTION
**Pull request by ScottMaclure**
Review welcome, many interesting choices to make!

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -74,6 +74,52 @@
...
+
+    <uxrow>
+        <uxcol class="large-12">
+            <h2>Pricing Table demo</h2>
+        </uxcol>
+    </uxrow>
+    <uxrow>
+        <uxcol class="large-6">
+
+            <uxpricingtable title="Samsung Galaxy S6" price="$699.99" href="/buy/samsunggalaxys6/now" status="comingsoon">
```
Comment (pvshum) [inline public/index.html:88]: @ScottMaclure  should we make **title** a separate component?

```
<uxtitle>
    Samsung Galaxy S6
</uxtitle>
```

to support HTMLs

Comment (ScottMaclure) [inline public/index.html:88]: Interesting again - do we deliberately limit the title to text? I'll go with your suggestion for now

Comment (pvshum) [inline public/index.html:88]: Let's stick with the plain text for now?

File: src/components/uxli/uxli.hbs [hunk 2]
```diff
@@ -0,0 +1 @@
+<li class="{{class}}">{{>content}}</li>
```
Comment (nathf) [inline src/components/uxli/uxli.hbs:1]: Have you both discussed the reasons against using `{{yield}}`?

Comment (pvshum) [inline src/components/uxli/uxli.hbs:1]: @ScottMaclure @nathf This is interesting, let's chat about it.

**Pull request merged**